### ISSUE TITLE
Add option to flip position components on prepareCanvas

### ISSUE THREAD

**Issue by luanpotter** [flame-engine/flame#64]
This is a very useful override of the prepareCanvas method that I've been using:

```dart
  // TODO pull up to flame! this is **very** useful
  @override
  void prepareCanvas(Canvas canvas) {
    canvas.translate(x, y);
    canvas.translate(width/2, height/2);
    canvas.scale(inverted ? 1.0 : -1.0, 1.0);
    canvas.translate(-width/2, -height/2);
  }
```

This flips horizontally. PositionComponent should have a field to control that and allow for easy flipping of sprites, both horizontally and vertically (or both).

It would be a really good to have feature.

**Comment by erickzanardo** [comment]
Fixed on #166

**Comment by erickzanardo** [comment]
Thanks @kurtome
